tests/providers/postgres/hooks/test_postgres.py

```diff
@@ -258,3 +258,16 @@ def test_insert_rows_replace_missing_replace_index_arg(self):
                 (2, "world",)]
         fields = ("id", "value")
         self.db_hook.insert_rows(table, rows, fields, replace=True)
+
+    @pytest.mark.backend("postgres")
+    def test_rowcount(self):
+        hook = PostgresHook()
+        input_data = ["foo", "bar", "baz"]
+
+        with hook.get_conn() as conn:
+            with conn.cursor() as cur:
+                cur.execute("CREATE TABLE {} (c VARCHAR)".format(self.table))
+                values = ",".join("('{}')".format(data) for data in input_data)
+                cur.execute("INSERT INTO {} VALUES {}".format(self.table, values))
+                conn.commit()
+                self.assertEqual(cur.rowcount, len(input_data))
```

Comment:
Please delete the table after test to avoid side effects 👌

Reply:
I believe that should be covered in `TestPostgresHook.tearDown` already.

Reply:
It is indeed!

Reply:
@cjsekl - I believe we also wanted to test if the log is written via mocking, rather than test if rowcount is in the cursor?

And since we are at that and it is added in dbapi_hook - maybe we can also add ver similar test in test_mysql/test_sqlite

Reply:
Sure thing. Remind me which log class I need to mock again?
I tried this, but not getting any calls to that log class.
```
class TestDbApiHook(unittest.TestCase):

    def setUp(self):
        super().setUp()

        self.cur = mock.MagicMock()
        self.conn = mock.MagicMock()
        self.conn.cursor.return_value = self.cur
        conn = self.conn

        class UnitTestDbApiHook(DbApiHook):
            conn_name_attr = 'test_conn_id'
            log = mock.MagicMock()

            def get_conn(self):
                return conn

        self.db_hook = UnitTestDbApiHook()

    def test_run_log(self):
        statement = 'SQL'
        assert self.db_hook.log.call_count == 2
```

Reply:
Sure thing. Just remind me which log class I need to mock again please?
I've tried this, but not getting any calls to that `log`.

```
class TestDbApiHook(unittest.TestCase):

    def setUp(self):
        super().setUp()

        self.cur = mock.MagicMock()
        self.conn = mock.MagicMock()
        self.conn.cursor.return_value = self.cur
        conn = self.conn

        class UnitTestDbApiHook(DbApiHook):
            conn_name_attr = 'test_conn_id'
            log = mock.MagicMock()

            def get_conn(self):
                return conn

        self.db_hook = UnitTestDbApiHook()

    def test_run_log(self):
        statement = 'SQL'
        self.db_hook.run(statement)
        assert self.db_hook.log.call_count == 2
```

Reply:
Ah, that should be `assert self.db_hook.log.info.call_count == 2`.
Is that what you had in mind?

Reply:
Yep